fly.toml

```diff
@@ -15,3 +15,7 @@ primary_region = "ord"
   auto_stop_machines = false
   auto_start_machines = true
   min_machines_running = 0
+
+[mounts]
```

Comment:
This adds a volume.  I already deployed to fly.io.  It's at /data, and I copied the files in there.  We'll have to do it again when you get this to deploy so we don't lose any moves.